test/performance/latency/main_LatencyTest.cpp

```diff
@@ -470,6 +470,18 @@ int main(
         }
     }
 
+    // Check parameters validity
+    if (use_security && test_agent == TestAgent::BOTH)
+    {
+        logError(LatencyTest, "Intra-process delivery NOT supported with security");
+        return 1;
+    }
```

Comment:
payload or RTPS encryption is not supported with DataSharing either. Should we check it here too?

Reply:
👍